spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/SpannerStatementQueryExecutor.java

```diff
@@ -83,7 +84,7 @@ else if (tree.isExistsProjection()) {
 	 * @throws IllegalArgumentException if the number of tags does not match the number of
 	 * params, or if a param of an unsupported type is given.
 	 */
-	public static Statement buildStatementFromSqlWithArgs(String sql, List<String> tags,
+	private static Statement buildStatementFromSqlWithArgs(String sql, List<String> tags,
```

Comment:
@balopat I had this as public because it will be used by the SQL annotated query method directly